js/activity.js

```diff
@@ -4501,6 +4500,8 @@ function Activity() {
                 " " +
                 document.body.clientHeight
         );
+        var newTitle = this.PlanetInterface.getCurrentProjectName;
```

Comment:
You can store the project name in this.planet and set the title when you return from the planet (but not here). Probably in this.showMusicBlocks